keyboards/darkproject/kd87b_ltd/readme.md

```diff
@@ -0,0 +1,25 @@
+# Dark Project KD87b LTD
+
+![Dark Project KD87b LTD](https://i.imgur.com/v33Gk5T.png)
```

Comment:
```suggestion
![Dark Project KD87b LTD](https://i.imgur.com/v33Gk5Th.png)
```

Reply:
The main idea is to use PNG's transparency, why you suggest to just use black background?

Reply:
From  [Imgur docs](https://api.imgur.com/models/image)
>  There are 6 total thumbnails that an image can be resized to. Each one is accessable by appending a single character suffix to the end of the image id, and before the file extension. The thumbnails are: 

> Thumbnail Suffix | Thumbnail Name | Thumbnail Size | Keeps Image Proportions
> h | Huge Thumbnail | 1024x1024 | Yes

Reply:
In your variant I'll get ugly black background. Why I should use thumbnail suffix? The original image is correctly displayed on keyboard page on different platforms with different scaling settings. You can check it in my fork branch.

Reply:
Okey, I reuploaded image with h suffix.